pointcloud/pointcloud_file_test.go

```diff
@@ -39,6 +39,7 @@ func TestNewFromFile(t *testing.T) {
 	nextCloud, err := NewFromFile(temp.Name(), logger)
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, nextCloud, test.ShouldResemble, cloud)
+	// want to add test for pcd here but not sure how to use artifact
```

Comment:
not sure how to use artifact because it looks like there's files cached somewhere I don't know how to add to that cache so a test I write won't fail on another computer